index.html

```diff
@@ -93,7 +93,7 @@ <h3> Title </h3>
             </div>
             <div class="film">
                 <div>
```

Comment:
Avoid lines of code that are too long (100 characters or more).

Reply:
- [ ] (Optional) It would be nice to suggest changes to the author of a code in a nice way rather than command or use an authoritative tone to suggest the change. For example, your comment can look like this _**" Kindly Avoid lines of code that are too long (100 characters or more). because this makes your code difficult to read and not clean"**_